test/runtests.jl

```diff
@@ -23,6 +22,10 @@ end
 
 include("TestFunctions.jl")
 
+for target in readdir("util")
+    include("util/$target")
+end
+
```

Comment:
Hmmn. Would be good if these could be added to the target list as `util/XXXX` so they behave just like seq, intervals etc on the command line. Would negate this for loop too.